test/kitchen/test/integration/common/rspec/spec_helper.rb

```diff
@@ -746,7 +746,10 @@ def is_file_signed(fullpath)
             'C:/Windows/ServiceProfiles/NetworkService/AppData/Local/Microsoft/Windows/DeliveryOptimization/Cache/',
             'C:/Windows/SoftwareDistribution/DataStore/Logs/',
             'C:/Windows/System32/wbem/Performance/',
-            'c:/windows/System32/LogFiles/'
+            'c:/windows/System32/LogFiles/',
+            'c:/windows/SoftwareDistribution/',
+            'c:/windows/ServiceProfiles/NetworkService/AppData/',
+            'c:/windows/System32/Tasks/Microsoft/Windows/UpdateOrchestrator'
```

Comment:
Should there be a `/` at the end here too? It's also a directory

Reply:
Done.